### DISCUSSION TITLE
Running in Home Assistant Docker container without network mode set to "host"

### DISCUSSION THREAD

**Discussion by DevJasperNL** [leikoilja/ha-google-home#835] [Q&A]
Hey guys,

The documentation states that "network_mode: host" should be used to get this integration to work if Home Assistant is running inside Docker, but I am wondering if anyone figured out if it is possible by mapping the correct ports.

I tried googling for the ports used by google home and added:
```
- "53:53/udp"
- "123:123/udp"
- "8008:8008/tcp"
- "8009:8009/tcp"
- "80:80/tcp"
- "443:443/tcp"
```

Sadly, no luck so far.

**Comment by gabbieblue** [comment]
Just wanted to ask if theres any updates on this? I saw there was a mention of implementing a solution in #774 but no PRs have been submitted for this.
Any luck with the workaround?
